clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -654,26 +654,26 @@ $ $ bin64/drrun -t drcachesim -simulator_type view -indir drmemtrace.*.dir -sim_
 Output format:
```

Comment:
If a PR corresponds to an issue, always include an `Issue: #nnnn` (or `Fixed: #nnnn`) line at the bottom of the PR description, and prefix the title with `i#nnnn: ` (or sthg like `i#nnnn descr, part N: ` if it is a planned-as-multi-step feature).  If you agree with me that the new issue you filed could be folded into the original it would be the original issue number here.